plugins/georeplication/gsyncd.go

```diff
@@ -3,17 +3,34 @@ package georeplication
 import (
 	"fmt"
 	"path"
+	"strings"
 
 	"github.com/gluster/glusterd2/glusterd2/gdctx"
+	"github.com/gluster/glusterd2/pkg/utils"
 	georepapi "github.com/gluster/glusterd2/plugins/georeplication/api"
 
 	config "github.com/spf13/viper"
 )
 
-const (
-	gsyncdCommand = "/usr/local/libexec/glusterfs/gsyncd"
+var (
+	defaultGsyncdCommand = "/usr/libexec/glusterfs/gsyncd"
+	gsyncdCommand        = ""
 )
 
+func getGsyncdCommand() string {
+	if gsyncdCommand != "" {
+		return gsyncdCommand
+	}
+
+	out, err := utils.ExecuteCommandOutput("glusterfsd", "--print-libexecdir")
+	if err != nil {
+		return defaultGsyncdCommand
```

Comment:
store this `defaultGsyncdCommand` in `gsyncdCommand` to avoid command execution in case of error reading `libexecdi`

Reply:
done